projects/packages/my-jetpack/_inc/components/my-jetpack-screen/index.jsx

```diff
@@ -135,8 +137,23 @@ export default function MyJetpackScreen() {
 		return null;
 	}
 
+	const resetOptionsMenuItem = {
+		label: _x(
+			'Reset Options (dev only)',
+			'Button for option to reset Jetpack Options',
+			'jetpack-my-jetpack'
+		),
+		title: __( 'Reset Options', 'jetpack-my-jetpack' ),
+		role: 'button',
+		onClick: () => resetJetpackOptions(),
+		onKeyDown: e => onKeyDownCallback( e, () => resetJetpackOptions() ),
+	};
+
 	return (
-		<AdminPage siteAdminUrl={ adminUrl }>
+		<AdminPage
+			siteAdminUrl={ adminUrl }
+			optionalMenuItems={ isDevVersion && userIsAdmin ? [ resetOptionsMenuItem ] : [] }
```

Comment:
I think this PR is missing a commit, the `AdminPage` component is not receiving `optionalMenuItems` as a prop and passing it down to `JetpackFooter` yet. With that, the option should appear

Reply:
No idea how that happened 😅